pkg/cable/libreswan/libreswan.go

```diff
@@ -272,14 +272,16 @@ func (i *libreswan) refreshConnectionStatus() error {
 			}
 		}
 
-		cable.RecordConnection(cableDriverName, &i.localEndpoint, &i.connections[j].Endpoint, string(i.connections[j].Status), false)
-		cable.RecordRxBytes(cableDriverName, &i.localEndpoint, &i.connections[j].Endpoint, rx)
-		cable.RecordTxBytes(cableDriverName, &i.localEndpoint, &i.connections[j].Endpoint, tx)
+		cable.RecordConnection(cableDriverName, &i.localEndpoint, &i.connections[j].Endpoint, string(i.connections[j].Status), false,
+			i.connections[j].GetFamily())
```

Comment:
`i.connections[j].GetFamily()` could go into a variable since it's called multiple times.